packages/client/tests/functional/issues/21352-id-does-not-exist/tests.ts

```diff
@@ -0,0 +1,45 @@
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+testMatrix.setupTestSuite(
+  () => {
+    // TODO: this currently fails with "The column `j1.id` does not exist in the current database.".
+    // Close https://github.com/prisma/prisma/issues/21352 once fixed.
+    test('[1] should not fail', async () => {
+      await prisma.relation1.findMany({
+        select: {
+          id: true,
+        },
+        where: {
+          user: {
+            email: 'info@example.com',
+          },
+        },
+      })
+    })
+
+    // TODO: this currently fails with "The column `j1.field` does not exist in the current database.".
+    // Close https://github.com/prisma/prisma/issues/21352 once fixed.
+    test('[2] should not fail', async () => {
```

Comment:
```suggestion
    test.failing('[2] should not fail', async () => {
```

Reply:
We can also wait for the engines to be merged

Reply:
> We can also wait for the engines to be merged
That'd be the plan, yeah.